problems/4.arrays.js

```diff
@@ -12,16 +12,18 @@ function firstAndLastAverage() {
 * @param {number[]} arr - The input array
 * @returns {number[]} - The duplicated array
 */
-function duplicateArray() {
-
-}
+// function duplicateArray(arr) {
+//   let arr = arr1;
+//   return arr.concat(arr1)
+// }
```

Comment:
This was close. Don't comment out the function definition or the specs will break. You just needed to do arr.concat(arr). You probably got an error saying arr1 not declared. or arr already declared. Your variable can't have the same name as your parameter, and can't have the value of some variable that hasn't been declared.